src/nv_ingest/extraction_workflows/pdf/pdfium_helper.py

```diff
@@ -148,45 +155,59 @@ def extract_table_and_chart_images(
         The original image from which objects were detected.
     page_idx : int
         The index of the current page being processed.
-    tables_and_charts : List[Tuple[int, ImageTable]]
-        A list to which extracted tables and charts will be appended.
+    page_elements : List[Tuple[int, ImageTable]]
+        A list to which extracted page elements will be appended.
 
     Notes
     -----
     This function iterates over detected objects, crops the original image to the bounding boxes,
-    and runs additional inference on the cropped images to extract detailed information about tables
-    and charts.
+    and runs additional inference on the cropped images to extract detailed information about page
+    elements.
 
     Examples
     --------
     >>> annotation_dict = {"table": [], "chart": []}
     >>> original_image = np.random.rand(1536, 1536, 3)
-    >>> tables_and_charts = []
-    >>> extract_table_and_chart_images(annotation_dict, original_image, 0, tables_and_charts)
+    >>> page_elements = []
+    >>> extract_page_element_images(annotation_dict, original_image, 0, page_elements)
     """
+    orig_width, orig_height, *_ = original_image.shape
+    pad_width, pad_height = padding_offset
 
-    width, height, *_ = original_image.shape
-    for label in ["table", "chart"]:
+    for label in ["table", "chart", "infographic"]:
         if not annotation_dict:
             continue
 
+        if label not in annotation_dict:
+            continue
+
         objects = annotation_dict[label]
+
         for idx, bboxes in enumerate(objects):
             *bbox, _ = bboxes
-            h1, w1, h2, w2 = bbox
+            w1, h1, w2, h2 = bbox
 
-            cropped = crop_image(original_image, (int(h1), int(w1), int(h2), int(w2)))
+            cropped = crop_image(original_image, (int(w1), int(h1), int(w2), int(h2)))
             base64_img = numpy_to_base64(cropped)
 
-            table_data = CroppedImageWithContent(
+            bbox_in_orig_coord = (
+                int(w1) - pad_width,
+                int(h1) - pad_height,
+                int(w2) - pad_width,
+                int(h2) - pad_height,
+            )
+            max_width = orig_width - 2 * pad_width
+            max_height = orig_height - 2 * pad_height
```

Comment:
This fixes a bug in bounding boxes in metadata. The cropped images are correct in `cropped = crop_image(original_image, (int(w1), int(h1), int(w2), int(h2)))` but `CroppedImageWithContent(bbox=...)` was incorrect. Some users might want to use this bbox coordinates to extract the images directly from the pdf instead of using the base64 image in `table_content` with something like:
```python
        w_orig = page.rect.width
        h_orig = page.rect.height
        loc = md["metadata"]["table_metadata"]["table_location"]        
        w_max, h_max = md["metadata"]["table_metadata"]["table_location_max_dimensions"]
        bbox = [loc[0] / w_max * w_orig, loc[1] / h_max * h_orig, loc[2] / w_max * w_orig, loc[3] / h_max * h_orig]
        pix = page.get_pixmap(clip=bbox)
```